src/main/java/com/google/cloud/genomics/utils/grpc/VariantUtils.java

```diff
@@ -38,6 +38,28 @@
   public static final String GATK_NON_VARIANT_SEGMENT_ALT = "<NON_REF>";
 
   /**
+   * Determines whether a variant represents a multi-nucleotide change.
+   */
+  public static final Predicate<Variant> IS_MULTI_NUCLEOTIDE = new Predicate<Variant>() {
+    @Override
+    public boolean apply(Variant variant) {
+      int refSize = variant.getReferenceBases().length();
+      if (refSize < 1) {
```

Comment:
I think you meant to say `==` since variants need to be at least a A, C, G, T, or N.